src/table/index.tsx

```diff
@@ -37,6 +42,12 @@ const Table = React.forwardRef(
         progressiveLoading: !!props.getLoadingStatus,
         inlineEdit: props.columnDefinitions.some(def => !!def.editConfig),
         disabledInlineEdit: props.columnDefinitions.some(def => !!def.editConfig?.disabledReason),
+        hasSortableColumns: props.columnDefinitions.some(def => def.sortingField || def.sortingComparator),
+        hasHiddenColumns,
+        hasStickyColumns,
+        hasFilterSlot: !!props.filter,
+        hasPaginationSlot: !!props.pagination,
+        itemCount: items.length,
```

Comment:
how useful is this? since this is the item count in current page and not the full item count

Reply:
I feel like this makes the data more unreliable, I wouldn't have a way to detrmine if this number is total items count or a page count